src/app/index.css

```diff
@@ -40,3 +48,7 @@ strong {
   }
 }
 
+/* Make card tagline grey */
+.ams-card__heading-group > .ams-paragraph--small {
+  color: #474747;
```

Comment:
Our custom properties are all available here, so you could use `var(--ams-brand-color-neutral-80)` (and then wait for the release)

Reply:
I’d rather not wait for the release. This stylesheet contains some application overrides; I don’t think we should keep it to the highest standards.